packages/web/src/pages/pay-and-earn-page/components/AudioCoinCard.tsx

```diff
@@ -16,14 +24,21 @@ export const AudioCoinCard = ({ onClick }: { onClick?: () => void }) => {
 
   const isLoading = isAudioBalanceLoading || isAudioPriceLoading
 
+  const handleCoinClick = useCallback(
```

Comment:
if this card is only for $AUDIO, why not just put `env.WAUDIO_MINT_ADDRESS` here in the callback so we avoid another function?